examples_utils/benchmarks/environment_utils.py

```diff
@@ -54,6 +34,11 @@ def check_poprun_env_variables(benchmark_name: str, cmd: str):
 
     """
 
+    # If PARTITION exists in env but IPUOF_VIPU_API_PARTITION_ID isnt, set it
+    # to the existing value
+    if (os.getenv("PARTITION") is not None) and (os.getenv("IPUOF_VIPU_API_PARTITION_ID") is None):
+        os.environ["IPUOF_VIPU_API_PARTITION_ID"] = os.getenv("PARTITION")
```

Comment:
```suggestion
        os.environ["IPUOF_VIPU_API_PARTITION_ID"] = os.environ["PARTITION"]
```

Reply:
Fair, by this point we've already determined it will return a value